content/community/depression.md

```diff
@@ -6,3 +6,130 @@ draft: false
 type: community
 kind: "health_and_food"
 ---
+
+[Purpose](purpose)  ||  [Channel Operators](ops) ||  [Rules & Guidelines](guidelines)  ||  [Ban Appeals](appeals)  || [Webchat](https://webchat.snoonet.org/depression) || [Links & Resources](resources)
```

Comment:
I'm not 100% certain but I'm pretty sure `[Purpose](purpose)` would try to link to a page called `purpose` in the current directory, it needs to start with `#` to link to an ID

Reply:
The # is not required, I tested it

Reply:
I just tested it and yes it is, clicking "Purpose" navigates to `http://localhost:1313/community/depression/purpose`